tripal_chado/includes/TripalImporter/OBOImporter.inc

```diff
@@ -1063,27 +1068,63 @@ class OBOImporter extends TripalImporter {
             '!url' => $full_url,
           ]), TRIPAL_WARNING);
       }
-      //What should happen with this stuff?
-      $base_iri = $ontology_results['config']['baseUris'][0];
-      $ontologyID = $ontology_results['ontologyId'];
-      $this->baseIRIs[$short_name] = [$ontologyID, $base_iri];
+      // The following foreach code works but, I am not sure that
+      // I am retrieving each query result from the json associative
+      // array with the correct style
+      foreach ($ontology_results['response']['docs'] as $each ) {
+        $obo_id = $each['obo_id'];
+        $defining_ontology = $each['is_defining_ontology'];
+        // First result should have defining_ontology=true, but if 
+        // it doesn't, use the first result with obo_id=$id
+	if ($defining_ontology == 'false' and $obo_id != $id) {
+	  continue;
+	}
```

Comment:
oops another indention issue.

Reply:
I delete the white space and tried again. I set up my .vimrc. No more crazy white spaces for me.